<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_spots_dialog</class>
 <widget class="QDialog" name="view_spots_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Spots</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="grain_id">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="grain_id_label">
     <property name="text">
      <string>Grain ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <spacer name="horizontal_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="detector_label">
     <property name="text">
      <string>Detector:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="peak_id_label">
     <property name="text">
      <string>Peak ID:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="hkl_label">
     <property name="text">
      <string>HKL:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QComboBox" name="peak_id">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLabel" name="tolerances_label">
     <property name="text">
      <string>Tolerances:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="hkl">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QLabel" name="tolerances">
     <property name="text">
      <string>0.25 tth, 3.0 eta, and 2.0 ome </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="13">
    <layout class="QVBoxLayout" name="canvas_layout"/>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="detector">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="show_ome_centers">
     <property name="text">
      <string>Show Omega Centers</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="3">
    <widget class="QCheckBox" name="show_frame_indices">
     <property name="text">
      <string>Show Frame Indices</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grain_id</tabstop>
  <tabstop>detector</tabstop>
  <tabstop>hkl</tabstop>
  <tabstop>peak_id</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
